### DISCUSSION TITLE
Camera Streaming example: camera stream starts after about 30 secs

### DISCUSSION THREAD

**Discussion by Paltr** [videokit-ai/videokit#112] [Q&A] [answered]
Hi!

I created a test scene similar you did in "Camera Streaming" example.
But video streaming starts with a big delay, about 40 seconds.
I recorded the video that demonstrates the problem:
https://github.com/natmlx/videokit/assets/6074214/fdc17a79-ad9a-471d-b5de-5d0000c91178

Here is the info about the system I'm running on:
OS Name	Microsoft Windows 11 Pro
Version	10.0.22631 Build 22631
Other OS Description 	Not Available
OS Manufacturer	Microsoft Corporation
System Name	DESKTOP-HQILK7R
System Manufacturer	AZW
System Model	GTR
System Type	x64-based PC
System SKU	Default string
Processor	AMD Ryzen 7 7840HS w/ Radeon 780M Graphics, 3801 Mhz, 8 Core(s), 16 Logical Processor(s)
BIOS Version/Date	American Megatrends International, LLC. GTRP01_P5C6V14, 13.06.2023
SMBIOS Version	3.5
Embedded Controller Version	255.255
BIOS Mode	UEFI
BaseBoard Manufacturer	AZW
BaseBoard Product	GTR
BaseBoard Version	V11
Platform Role	Desktop
Secure Boot State	Off
PCR7 Configuration	Elevation Required to View
Windows Directory	C:\WINDOWS
System Directory	C:\WINDOWS\system32
Boot Device	\Device\HarddiskVolume1
Hardware Abstraction Layer	Version = "10.0.22621.2506"
User Name	DESKTOP-HQILK7R\Paltr
Installed Physical Memory (RAM)	32,0 GB
Total Physical Memory	27,7 GB
Available Physical Memory	6,96 GB
Total Virtual Memory	54,7 GB
Available Virtual Memory	11,5 GB
Page File Space	27,0 GB
Page File	C:\pagefile.sys
Kernel DMA Protection	Off
Virtualization-based security	Not enabled
Windows Defender Application Control policy	Enforced
Windows Defender Application Control user mode policy	Off
Device Encryption Support	Elevation Required to View
Hyper-V - VM Monitor Mode Extensions	Yes
Hyper-V - Second Level Address Translation Extensions	Yes
Hyper-V - Virtualization Enabled in Firmware	Yes
Hyper-V - Data Execution Protection	Yes

**Comment by olokobayusuf** [comment]
Hey there @Paltr, what is the model of your webcam?

**Comment by Paltr** [comment]
HD Pro Webcam C920. This behavior doesn't reproduce with the standard WebCamTexture - the image appears instantly.

**Reply by olokobayusuf** [reply, marked as answer]
Hey @Paltr this bug should be fixed in the latest alpha build. Try it out by upgrading to VideoKit 0.0.18-alpha.8.
